dev-release

```diff
@@ -61,12 +61,12 @@ const writePkg = async (pkg, json) => {
   console.log("5. Build design & login SDKs");
```

Comment:
Temporarily disabling is needed for login-sdk.
Both Good-Design / Login-Sdk were not yet updated to latest version which has a hard requirement for typescript >5 (mentioned 2 weeks ago)

Good-Design I now tested and seems to work.

Login-sdk has to be retested but I did do this yet

Reply:
pls add some cmdline flag to skip login package